components/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt/src/main/java/org/wso2/carbon/identity/api/resource/mgt/dao/impl/APIResourceManagementDAOImpl.java

```diff
@@ -1188,4 +1256,25 @@ private void lessThanFilterBuilder(int count, String value, String attributeName
         filter.append(attributeName).append(filterString);
         filterQueryBuilder.setFilterAttributeValue(count, value);
     }
+
+    private boolean isInternalFiltering(FilterQueryBuilder filterQueryBuilder) {
+
+        if (filterQueryBuilder.getFilterAttributeValue() != null) {
+            return filterQueryBuilder.getFilterAttributeValue().get(1).contains("internal") ||
+                    filterQueryBuilder.getFilterAttributeValue().get(1).contains("console");
```

Comment:
Don't we have an existing constant for internal and console?

Reply:
Removed the logic with the new implementation.